src/box/lua/schema.lua

```diff
@@ -785,6 +793,14 @@ local function update_index_parts(format, parts)
             box.error(box.error.ILLEGAL_PARAMS,
                       "options.parts[" .. i .. "]: type (boolean) is expected")
         end
+        if (not part.is_nullable) and part.exclude_null then
+            if part.is_nullable ~= nil then
+                box.error(box.error.ILLEGAL_PARAMS,
+                       "options.parts[" .. i .. "]: exclude_null=true and " ..
+                       "is_nullable=false are contradictory")
```

Comment:
I'd say 'incompatible' (still personally I believe that they should be compatible).

Reply:
fixed